pkg/haprovider/haprovider.go

```diff
@@ -96,6 +97,17 @@ func (r *HAProvider) createService(
 		return nil, err
 	}
 
+	// Get cluster primary ip family, which is used for HA service
+	primaryIPFamily, err := utils.GetPrimaryIPFamily(cluster)
+	if err != nil {
+		return nil, err
+	}
+	if primaryIPFamily == "V6" {
+		primaryIPFamily = "IPv6"
```

Comment:
Can we make all those string const variables, we still need to use them in future